test/jit/test_freezing.py

```diff
@@ -1517,3 +1517,36 @@ def test_optimize_freeze_module(self):
         # optimize_frozen_module should be run
         frozen_mod = torch.jit.freeze(torch.jit.script(mod.eval()))
         FileCheck().check_not("batch_norm").run(frozen_mod.graph)
+
+    def test_freeze_remove_dropout(self):
+        class Net(nn.Module):
+            def __init__(self):
+                super(Net, self).__init__()
+                self.conv1 = nn.Conv2d(1, 32, 3, 1)
+                self.conv2 = nn.Conv2d(32, 64, 3, 1)
```

Comment:
Nit: the conv and linear here are quite big - they could be smaller without effecting the test and it would run quicker (if we run the model...). I would also consider making the module simpler / remove some layers to parse the test easier.